harpy/_misc.py

```
@@ -73,26 +101,38 @@ def gzip_file(infile):
     def convert(self, value, param, ctx):
         try:
-            parts = value.split(',')
-            if len(parts) != 2:
-                self.fail(f"{value} is not a valid int pair. The value should be two integers separated by a comma.", param, ctx)
+            parts = [i.strip() for i in value.split(',')]
+            if len(parts) != self.max_entries:
+                raise ValueError
+            for i in parts:
+                try:
+                    int(i)
+                except:
```

Ruff

Do not use bare `except`

(E722)